### DISCUSSION TITLE
JPA 3.x support

### DISCUSSION THREAD

**Discussion by hantsy** [micronaut-projects/micronaut-data#2024] [Q&A]
I am trying to update to use JPA 3.x (aka jakarta namespace), found `data-jpa` does not support it.
https://github.com/hantsy/micronaut-sandbox/actions/runs/4194409301/jobs/7272532780

But other examples, jdbc with `Specification`, it is ok to use the JPA 3.x, https://github.com/hantsy/micronaut-sandbox/blob/master/data-jdbc-kotlin/build.gradle.kts#L58, and the Specification example: https://github.com/hantsy/micronaut-sandbox/blob/master/data-jdbc-kotlin/src/main/kotlin/com/example/Specifications.kt

**Comment by dstepanov** [comment]
JPA 3 is only supported with Hibernate 6, coming in Micronaut 4
